doc/explanation/dockstore.md

```diff
@@ -119,6 +121,11 @@ We reserve path prefixes to support mounting user and data (fuse, read-only) dat
 * `${USER_VOLUME}` mounts the per-user persistent storage folder
 * `${DATA_VOLUME}` mounts the read-only `gen3-fuse` proxy to the commons objects referenced by the workspace manifest
 
+There is also a possibility to tweak the User ID or filesystem GID to in order to let the user to have correct access permissions to mounted volumes. There are two config fields for this purpose, noted as below. They should be places in the first container's (root service) config YAML file (see `example 2`)
+
+* `user_uid` the UID for the user in this container.
+* `fs_gid` the GID for the filesystem mounts.
```

Comment:
should "group_uid" be in the doc too?

Reply:
I thinked about it, but we didn't advertise "group_uid" in the JSON format configs, don't know if that is intentional 🤔